leapp/cli/__init__.py

```diff
@@ -7,9 +7,14 @@
 
 
 @command('')
-@command_opt('debug', is_flag=True, help='Enable debug logging', inherit=True)
+@command_opt('debug', is_flag=True, help='Enable debug mode', inherit=True)
+@command_opt('verbose', is_flag=True, help='Enable verbose logging', inherit=True)
 def cli(args):
     os.environ['LEAPP_DEBUG'] = '1' if args.debug else os.environ.get('LEAPP_DEBUG', '0')
+    if os.environ['LEAPP_DEBUG'] == '1' or args.verbose:
```

Comment:
Aren't L14-17 equivalent to 
os.environ['LEAPP_VERBOSE'] = '1' if os.environ['LEAPP_DEBUG'] == '1' or args.verbose else os.environ('LEAPP_VERBOSE', '0')?

Just to be consistent with LEAPP_DEBUG setting at L13

Reply:
The first version I used one line, but it has broken the CI because it has a rule "line too long > 120"

Reply:
Why not split long line into 2 instead of modifying the logic because of line too long?